cmd/podman/containers/restart.go

```diff
@@ -47,20 +48,35 @@ var (
 )
 
 var (
-	restartOptions = entities.RestartOptions{}
-	restartTimeout uint
+	restartOpts = entities.RestartOptions{
+		Filters: make(map[string][]string),
+	}
+	restartCidFiles = []string{}
+	restartTimeout  uint
 )
 
 func restartFlags(cmd *cobra.Command) {
 	flags := cmd.Flags()
 
-	flags.BoolVarP(&restartOptions.All, "all", "a", false, "Restart all non-running containers")
-	flags.BoolVar(&restartOptions.Running, "running", false, "Restart only running containers when --all is used")
+	flags.BoolVarP(&restartOpts.All, "all", "a", false, "Restart all non-running containers")
+	flags.BoolVar(&restartOpts.Running, "running", false, "Restart only running containers when --all is used")
```

Comment:
Maybe it's my end of week brain not clicking, but I don't see a requirement that the `--all` option is needed with the `running` option?  Is it required?  If so, the man page needs to reflect that.  However, I think it's not required, so I would simply change this to:

```suggestion
	flags.BoolVar(&restartOpts.Running, "running", false, "Restart only running containers")
```